packages/driver/src/cy/commands/navigation.ts

```diff
@@ -95,24 +95,16 @@ const specifyFileByRelativePath = (url, log) => {
 }
 
 const aboutBlank = (cy, win) => {
-  currentlyVisitingAboutBlank = true
-
   if (Cypress.state('isMultiDomain')) {
     return new Promise((resolve) => {
-      Cypress.specBridgeCommunicator.once('visit:about:blank:end', () => {
-        currentlyVisitingAboutBlank = false
-        resolve()
-      })
+      Cypress.specBridgeCommunicator.once('visit:about:blank:end', resolve)
```

Comment:
Do we need to put the `specBridgeCommunicator` on big Cypress? Can we use `Cypress.emit` and then in the multi domain code listen for that event and then leverage the `specBridgeCommunicator` in scope? I feel like the global communicators are already a bit of an anti pattern and hoping we can avoid it.

Reply:
This change was a result of this [comment](https://github.com/cypress-io/cypress/pull/20467#discussion_r819609707) along with a post-scrum discussion we had. We are trying to move the multi-domain events to its own communicator instead of using `Cypress.emit`. There is also a desire to combine both the primary and spec bridge communicators so there is a single multi-domain communicator.

Reply:
Definitely makes sense. Hopefully we can consolidate the communicator instances in the future so the communicator is somewhat domain agnostic